### DISCUSSION TITLE
Debugging Values is not shown on mouse hover

### DISCUSSION THREAD

**Discussion by rioasmara** [Vector35/binaryninja-api#5400] [General] [answered]
Hi.

I am using the latest free version of Binja. I am just reviewing how binja works with Debugger. While I am working with Debugger, I cannot see the current value of the variable with mouse hover 

I have initialized variable like below. The variable is initialized with value 1. 

![image](https://github.com/Vector35/binaryninja-api/assets/17211692/b0e01367-1f5b-41ab-a1c1-5b966bcdb667)

However, When the mouse is hovered on the variable while debuggin stopped at breakpoint. I cannot see the current value of the variable 

![image](https://github.com/Vector35/binaryninja-api/assets/17211692/63d54a56-164c-4bc6-97e2-b4ccc301c4cf)

I am just starting to use Binja lately, I came from IDA background.

**Comment by xusheng6** [comment, marked as answer]
We already have an issue for it https://github.com/Vector35/debugger/issues/106

**Comment by xusheng6** [comment]
Meanwhile, you can use menu -> "Debugger" -> "Create Stack View" to create a pane that shows the bytes on the stack, and view the content of the variable in it. I know this is inconvenient, and we will try to fix it sooner
